contracts/votes/src/balance.rs

```diff
@@ -57,64 +57,68 @@ pub fn mint_balance(e: &Env, to: &Address, amount: i128) {
 /// ### Panics
 /// This function panics if the balance is less than the amount to remove.
 pub fn burn_balance(e: &Env, from: &Address, amount: i128) {
-    let balance = storage::get_balance(e, from);
-    if balance < amount {
-        panic_with_error!(e, TokenVotesError::BalanceError);
-    }
+    if amount > 0 {
```

Comment:
Why don't we just panic if not over 0

Reply:
we panic if its less than 0, current the implemented native token contract does not panic if called with a transfer of 0, so this allows 0 to be input but won't actually do anything.